server/routes/hikes.js

```diff
@@ -39,7 +39,7 @@ router.get('/favs', (req, res) => {
     });
 });
 
-router.update('/favs', (req, res) => {
+router.get('/favs', (req, res) => {
```

Comment:
I see why update isn't correct but here should be 'put' instead of 'get'. I can fix post this approval